### PR TITLE
add default for authTitle

### PR DESCRIPTION
**Pull request by U5B**

### CODE REVIEW

**Comment by u9g** [comment]
please run `npm run fix`

**Comment by U5B** [comment]
> please run `npm run fix`

linted.

should this go into prismarine-auth instead as a default?

File: src/client/microsoftAuth.js [hunk 1]
```diff
@@ -8,6 +8,10 @@ async function authenticate (client, options) {
     options.profilesFolder = path.join(minecraftFolderPath, 'nmp-cache')
   }
 
+  if (!options.authTitle) {
```
Comment (extremeheat) [inline src/client/microsoftAuth.js:11]: Make this check === undefined, in pauth false is also allowed

**Comment by rom1504** [comment]
@extremeheat does that look correct?

**Comment by extremeheat** [comment]
Lgtm

**Pull request merged**